.goreleaser.yml

```diff
@@ -1,58 +1,59 @@
+project_name: botkube
 before:
   hooks:
     - go mod download
 builds:
-- id: botkube
-  binary: botkube
-  main: cmd/botkube/main.go
-  ldflags: &ldflags
-  - -s -w
-    -X github.com/infracloudio/botkube/pkg/version.Version={{ .Tag }}
-    -X github.com/infracloudio/botkube/pkg/version.GitCommitID={{ .Commit }}
-    -X github.com/infracloudio/botkube/pkg/version.BuildDate={{ .Date }}
-  env:
-  - CGO_ENABLED=0
-  goos:
-    - linux
-  goarch:
-    - amd64
-    - arm
-    - arm64
-  goarm:
-    - 7
+  - id: botkube
+    binary: botkube
+    main: cmd/botkube/main.go
+    ldflags: &ldflags
+      - -s -w
+        -X github.com/infracloudio/botkube/pkg/version.Version={{ .Tag }}
+        -X github.com/infracloudio/botkube/pkg/version.GitCommitID={{ .Commit }}
+        -X github.com/infracloudio/botkube/pkg/version.BuildDate={{ .Date }}
+    env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+    goarch:
+      - amd64
+      - arm
+      - arm64
+    goarm:
+      - 7
```

Comment:
@krunalhinguu is this change necessary?

Reply:
should i delete??

Reply:
i thought you told me to raise the PR

Reply:
or you talking about autofomating?? @PrasadG193

Reply:
Yes, I was talking about the formatting changes. Are those intentional? Does exiting format causing any issues?